client/views/events/edit/event.edit.js

```diff
@@ -263,9 +267,10 @@ Template.eventEdit.events({
 
 		var updateReplicas = instance.$("input[name='updateReplicas']").is(':checked');
 		var sendNotification = instance.$(".js-check-notify").is(':checked');
+		var addNotificationMessage = instance.$("#eventEditAddMessage").val();
 
 		instance.saving(true);
-		Meteor.call('saveEvent', eventId, editevent, updateReplicas, sendNotification, function(error, eventId) {
+		Meteor.call('saveEvent', eventId, editevent, updateReplicas, sendNotification, addNotificationMessage, function(error, eventId) {
```

Comment:
This parameter list is getting long. At some point we'll want to bundle the flags in an `options` parameter. Still manageable now though.